### ISSUE TITLE
Fox 9 Minneapolis (KMSP) 

### ISSUE THREAD

**Issue by jhedfors** [iptv-org/iptv#4578]
**_Channel Name:_** Fox 9 Minneapolis (KMSP)
**_Country:_** USA
**_Broken Link (from playlist):_** https://lnc-kmsp-fox-aws.tubi.video/index.m3u8
**_Possible Replacement (optional):_** xxx
**_Notes (optional):_** stream is repeating 10 second loop from a couple is days ago.

**Comment by jkemp814** [comment]
I can confirm. Plays for about nine seconds, then freezes and stops. Tested in Firefox, Kodi, and VLC.

**Comment by jkemp814** [comment]
None of the links are working on their website either.
https://www.fox9.com/live

**Comment by Sphinxroot** [comment]
yes tubi link  is looping  but  https://www.fox9.com/live  i get https://dcs-live.apis.anvato.net/server/play/p0dr1kQiGl4wtzj1/manifest.m3u8 +token   maybe not 24/7 it need to be streamlinked

**Comment by LaneSh4d0w** [comment]
> 
> 
> yes tubi link is looping but https://www.fox9.com/live i get https://dcs-live.apis.anvato.net/server/play/p0dr1kQiGl4wtzj1/manifest.m3u8 +token maybe not 24/7 it need to be streamlinked

https://iptv-all.lanesh4d0w.codes/united-states/fox-minneapolis
